ingress/controllers/nginx-third-party/README.md

```diff
@@ -426,6 +426,16 @@ Please check the example `rc-custom-configuration.yaml`
 
 If the Configmap it is updated, NGINX will be reloaded with the new configuration
 
+### NGINX status page
+
```

Comment:
please add a section with how to deploy the controller, i.e choose one rc.yaml as the main one, move it to the top level directory, and ask people to create that one before starting any of the examples.